src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -353,4 +353,36 @@ def load_arguments(self, _):
                         "all monitor settings would be cloned instead of expanding its scope.")
         c.argument('monitor_types', options_list=['--types', '-t'], arg_type=get_enum_type(['metricsAlert']),
                    nargs='+', help='List of types of monitor settings which would be cloned.', default=['metricsAlert'])
+
+    # region Private Link Resources
+    for item in ['create', 'update', 'show', 'delete', 'list']:
+        with self.argument_context('monitor private-link-scope {}'.format(item)) as c:
+            c.argument('scope_name', options_list=['--name', '-n'], help='Name of the Azure Monitor Private Link Scope.')
+    with self.argument_context('monitor private-link-scope create') as c:
+        c.ignore('location')
+
+    with self.argument_context('monitor private-link-scope scoped-resource') as c:
+        c.argument('scope_name', help='Name of the Azure Monitor Private Link Scope.')
+        c.argument('resource_name', options_list=['--name', '-n'], help='Name of the assigned resource.')
+        c.argument('linked_resource_id', options_list=['--linked-resource'], help='Name or ID of the linked resource. It should be one of log analytics workspace or application insights component.')
+
+    with self.argument_context('monitor private-link-scope private-link-resource') as c:
+        c.argument('scope_name', help='Name of the Azure Monitor Private Link Scope.')
+        c.argument('group_name', options_list=['--name', '-n'], help='Name of the private link resource.')
```

Comment:
>group_name [](start = 20, length = 10)

why is it named as gropu_name?

Reply:
For users, it's just `--name/-n`. I choose group_name here since there is a group id in each private link resource.